cmd/podman/utils/utils.go

```diff
@@ -139,3 +140,11 @@ func IsCheckpointImage(ctx context.Context, namesOrIDs []string) (bool, error) {
 	}
 	return true, nil
 }
+
+func RemoveSlash(input []string) []string {
+	var output []string
```

Comment:
Can initialize this with `make([]string, 0, len(input))` to ensure we only do one allocation